packages/directlinespeech/__tests__/sendSpeechActivity.js

```diff
@@ -18,8 +18,30 @@ beforeEach(() => {
   global.AudioContext = MockAudioContext;
 });
 
-test('should echo back when saying "hello" and "world"', async () => {
-  const { directLine, sendTextAsSpeech } = await createTestHarness();
+test('should echo back when saying "hello" and "world" with enableInternalHTTPSupport set to true', async () => {
+  const { directLine, sendTextAsSpeech } = await createTestHarness({ enableInternalHTTPSupport: true });
+
+  const connectedPromise = waitForConnected(directLine);
+  const activitiesPromise = subscribeAll(take(directLine.activity$, 2));
+
+  await connectedPromise;
+
+  await sendTextAsSpeech('hello');
+  await sendTextAsSpeech('world');
+
+  const activities = await activitiesPromise;
+  const activityUtterances = Promise.all(activities.map(activity => recognizeActivityAsText(activity)));
+
+  await expect(activityUtterances).resolves.toMatchInlineSnapshot(`
+    Array [
+      "Hello.",
+      "World.",
+    ]
+  `);
+});
+
+test('should echo back when saying "hello" and "world" with enableInternalHTTPSupport set to false', async () => {
+  const { directLine, sendTextAsSpeech } = await createTestHarness({ enableInternalHTTPSupport: false });
```

Comment:
If we set the default value for the options parameter to an empty object in the function definition, then we don't need to pass the options as a parameter to `createTestHarness` and `enableInternalHTTPSupport` will inherently be falsy. 

```suggestion
  const { directLine, sendTextAsSpeech } = await createTestHarness();
```